src/App/Application.cpp

```diff
@@ -50,6 +50,7 @@
 #include <QFileInfo>
 #include <QProcessEnvironment>
 #include <QStandardPaths>
+#include <SMESH_Version.h>

```

Comment:
This will cause a build failure if FreeCAD is built without SMESH support because then a SMESH_Version.h won't be generated. SMESH support is off if Fem and MeshPart is disabled. The recommended way is to include the file in LibraryVersions.h depending on the variable BUILD_SMESH

Reply:
Thanks. I changed the PR accordingly. Is this now correct?